### PR TITLE
Bug 1905115: Restart libvirt to detect we are using firewalld

### PR DESCRIPTION
**Pull request by stbenjam**
CI is currently failing with error below. This is a new problem, but
seems to be because libvirt is trying to use iptables since it was
started before firewalld. Moving the 02 stuff around to make sure we
always restart libvirtd after firewalld.

```
Executing: /usr/lib/systemd/systemd-sysv-install enable network
+(./02_configure_host.sh:168): main(): '[' y == y ']'
+(./02_configure_host.sh:169): main(): sudo virsh net-destroy ostestbm
Network ostestbm destroyed
+(./02_configure_host.sh:170): main(): sudo virsh net-start ostestbm
error: Failed to start network ostestbm
error: internal error: Failed to apply firewall rules /usr/sbin/iptables
-w --table filter --insert LIBVIRT_INP --in-interface ostestbm
--protocol tcp --destination-port 67 --jump ACCEPT: iptables: No
chain/target/match by that name.
```

### CODE REVIEW

File: 02_configure_host.sh [hunk 1]
```diff
@@ -85,13 +85,19 @@ if [ ${NUM_EXTRA_WORKERS} -ne 0 ]; then
   jq "{nodes: .nodes[-${NUM_EXTRA_WORKERS}:]}" ${ORIG_NODES_FILE} | tee ${EXTRA_NODES_FILE}
 fi
 
+ZONE="\nZONE=libvirt"
```
Comment (dtantsur) [inline 02_configure_host.sh:88]: nit: it wasn't necessary to move this bit

File: 02_configure_host.sh [hunk 2]
```diff
@@ -85,13 +85,19 @@ if [ ${NUM_EXTRA_WORKERS} -ne 0 ]; then
...
+sudo systemctl enable --now firewalld
+
 # Allow local non-root-user access to libvirt
 # Restart libvirtd service to get the new group membership loaded
 if ! id $USER | grep -q libvirt; then
   sudo usermod -a -G "libvirt" $USER
```
Comment (dtantsur) [inline 02_configure_host.sh:94]: Btw this won't work, it requires a `newgrp` call to take effect.

**Review by dtantsur — APPROVED** [review]

File: 02_configure_host.sh (hunk 2 above)
Comment (stbenjam) [reply 02_configure_host.sh:94]: If it doesn't work it's unrelated to the CI problem, but the restart of the libvirt daemon based on the comment seems to be enough.  The libvirt daemon verifies the users group membership I guess, not virsh running in the local shell.

Comment (dtantsur) [reply 02_configure_host.sh:94]: Quite possibly. But yeah, it's unrelated here.

**Review by dtantsur — APPROVED** [review]
/lgtm

**Pull request merged**